extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -601,6 +613,22 @@ class AmpAccordion extends AMP.BaseElement {
       tryFocus(newFocusHeader);
     }
   }
+
+  /**
+   * Callback function to execute when mutations are observed.
+   * @param {!Array<!MutationRecord>} mutations
+   */
+  toggleMutations_(mutations) {
+    mutations.forEach(mutation => {
+      if (mutation.attributeName === 'expanded') {
+        const sectionEl = dev().assertElement(mutation.target);
+        const toExpand = sectionEl.getAttribute('expanded');
+        if (toExpand === 'false') {
```

Comment:
Let's add a comment explaining why we decided to use the string value `"false"` and not the boolean `false`. It looks a little unusual without any explanation

Reply:
The `"false"` is more of a compiler formality, since in manual testing the condition worked fine with boolean `false`. Is there a reason to silence that type check to restore the more expected convention, or is it better to let this look funny and leave a comment?